src/unix/linux_like/mod.rs

```diff
@@ -267,7 +267,12 @@ cfg_if! {
                 pub stx_mnt_id: crate::__u64,
                 pub stx_dio_mem_align: crate::__u32,
                 pub stx_dio_offset_align: crate::__u32,
-                __statx_pad3: [crate::__u64; 12],
+                pub stx_subvol: crate::__u64,
+                pub stx_atomic_write_unit_min: crate::__u32,
+                pub stx_atomic_write_unit_max: crate::__u32,
+                pub stx_atomic_write_segments_max: crate::__u32,
+                __statx_pad2: [crate::__u32; 1],
+                __statx_pad3: [crate::__u64; 9],
```

Comment:
I'm dropping the new constant for now, as they seem likely to cause more headaches